### DISCUSSION TITLE
A More Robust Table View Of Songs

### DISCUSSION THREAD

**Discussion by Menghini** [audioling/audioling#6] [Ideas]
Like most implementations of tables, I can click on various things such as "#", "Title", "Duration", "Favorite", "Plays", etc. This was a feature of Sonixd and somehow slipped out of Feishin. My request is that when redesigning the app, to include features that allow someone to click on headers to sort in a more intuitive way.

Yes, I know it is a toggle above the table in Feishin. But why? That's more clicks.

**Comment by jeffvli** [comment]
Will be added
